code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -184,3 +184,27 @@
 			message = replacetextEx(message,regex(capitalize(key),"g"), "[capitalize(value)]")
 			message = replacetextEx(message,regex(key,"g"), "[value]")
 	speech_args[SPEECH_MESSAGE] = trim(message)
+
+/obj/item/clothing/mask/madvillainsmask
+	name = "the Madvillain's mask"
```

Comment:
did you test this ingame because this wont work like you think itll work

Reply:
I did before the pr. it just says smth like "you throw madvillains mask!"

Reply:
![image](https://user-images.githubusercontent.com/20987591/103720057-c4e0ff80-4fca-11eb-8f47-229b2d3643fe.png)

Reply:
![image](https://user-images.githubusercontent.com/20987591/103720120-ee019000-4fca-11eb-83d9-7e3b5d82b566.png)

Reply:
Oh okay I get what you're saying